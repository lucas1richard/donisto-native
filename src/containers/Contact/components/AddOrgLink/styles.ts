import { StyleSheet } from 'react-native';
import theme from 'theme/iftheme';

const styles = StyleSheet.create({
  wrapper: {
    backgroundColor: '#fff',
    justifyContent: 'center',
    alignItems: 'center',
    borderRadius: 12,
    overflow: 'hidden',
    marginRight: theme.screenPadding
  }
});

export default styles;