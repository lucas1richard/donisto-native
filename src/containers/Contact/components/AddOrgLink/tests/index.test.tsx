import * as React from 'react';
import { shallow } from 'enzyme';
import AddOrgLink from '..';

describe('<AddOrgLink />', () => {
  const renderComponent = () => shallow(<AddOrgLink />);
  it('renders with just text', () => {
    expect(renderComponent()).toMatchSnapshot();
  });
});