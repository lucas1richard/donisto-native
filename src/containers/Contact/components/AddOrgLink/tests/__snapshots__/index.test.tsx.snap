// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AddCauseLink /> renders with just text 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <AddCauseLink />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "activeOpacity": 0.2,
      "children": <Component
        style={
          Array [
            Object {
              "height": 100,
              "marginRight": 12,
              "width": 100,
            },
            Object {
              "alignItems": "center",
              "backgroundColor": "#fff",
              "borderRadius": 12,
              "justifyContent": "center",
              "marginRight": 12,
              "overflow": "hidden",
            },
          ]
        }
      >
        <Component
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "rgba(0,0,0,0.6)",
              "height": "100%",
              "justifyContent": "center",
              "width": "100%",
            }
          }
        >
          <Icon
            allowFontScaling={false}
            color="#fff"
            name="plus"
            size={50}
            style={
              Object {
                "marginRight": 5,
              }
            }
          />
        </Component>
      </Component>,
      "onPress": [Function],
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <Component
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "rgba(0,0,0,0.6)",
              "height": "100%",
              "justifyContent": "center",
              "width": "100%",
            }
          }
        >
          <Icon
            allowFontScaling={false}
            color="#fff"
            name="plus"
            size={50}
            style={
              Object {
                "marginRight": 5,
              }
            }
          />
        </Component>,
        "style": Array [
          Object {
            "height": 100,
            "marginRight": 12,
            "width": 100,
          },
          Object {
            "alignItems": "center",
            "backgroundColor": "#fff",
            "borderRadius": 12,
            "justifyContent": "center",
            "marginRight": 12,
            "overflow": "hidden",
          },
        ],
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <Icon
            allowFontScaling={false}
            color="#fff"
            name="plus"
            size={50}
            style={
              Object {
                "marginRight": 5,
              }
            }
          />,
          "style": Object {
            "alignItems": "center",
            "backgroundColor": "rgba(0,0,0,0.6)",
            "height": "100%",
            "justifyContent": "center",
            "width": "100%",
          },
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "allowFontScaling": false,
            "color": "#fff",
            "name": "plus",
            "size": 50,
            "style": Object {
              "marginRight": 5,
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": [Function],
      },
      "type": [Function],
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "activeOpacity": 0.2,
        "children": <Component
          style={
            Array [
              Object {
                "height": 100,
                "marginRight": 12,
                "width": 100,
              },
              Object {
                "alignItems": "center",
                "backgroundColor": "#fff",
                "borderRadius": 12,
                "justifyContent": "center",
                "marginRight": 12,
                "overflow": "hidden",
              },
            ]
          }
        >
          <Component
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "rgba(0,0,0,0.6)",
                "height": "100%",
                "justifyContent": "center",
                "width": "100%",
              }
            }
          >
            <Icon
              allowFontScaling={false}
              color="#fff"
              name="plus"
              size={50}
              style={
                Object {
                  "marginRight": 5,
                }
              }
            />
          </Component>
        </Component>,
        "onPress": [Function],
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <Component
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "rgba(0,0,0,0.6)",
                "height": "100%",
                "justifyContent": "center",
                "width": "100%",
              }
            }
          >
            <Icon
              allowFontScaling={false}
              color="#fff"
              name="plus"
              size={50}
              style={
                Object {
                  "marginRight": 5,
                }
              }
            />
          </Component>,
          "style": Array [
            Object {
              "height": 100,
              "marginRight": 12,
              "width": 100,
            },
            Object {
              "alignItems": "center",
              "backgroundColor": "#fff",
              "borderRadius": 12,
              "justifyContent": "center",
              "marginRight": 12,
              "overflow": "hidden",
            },
          ],
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <Icon
              allowFontScaling={false}
              color="#fff"
              name="plus"
              size={50}
              style={
                Object {
                  "marginRight": 5,
                }
              }
            />,
            "style": Object {
              "alignItems": "center",
              "backgroundColor": "rgba(0,0,0,0.6)",
              "height": "100%",
              "justifyContent": "center",
              "width": "100%",
            },
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "allowFontScaling": false,
              "color": "#fff",
              "name": "plus",
              "size": 50,
              "style": Object {
                "marginRight": 5,
              },
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": [Function],
        },
        "type": [Function],
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": false,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
